analyses/Data_ingest.R

```diff
@@ -0,0 +1,108 @@
+base_paths <- model_file_dict(params$run_id, params$run_id_year)
+comparison_paths <- model_file_dict(
+  params$comparison_run_id,
+  params$comparison_run_id_year
+)
+run_id <- params$run_id
+comparison_run_id <- params$comparison_run_id
+
+analyses_paths <- list(
+  output = list(
+    list(
+      s3 = base_paths$output$assessment_card$s3,
+      key = "assessment_card"
+    ),
+    list(
+      s3 = base_paths$output$assessment_pin$s3,
+      key = "assessment_pin"
+    ),
+    list(
+      s3 = base_paths$output$metadata$s3,
+      key = "metadata"
+    ),
+    list(
+      s3 = base_paths$output$performance_test$s3,
+      key = "performance_test"
+    ),
+    list(
+      s3 = base_paths$output$shap$s3,
+      key = "shap"
+    )
+  )
+)
+
+source("analyses/Ingest_script.qmd")
+data_new <- model_fetch_run_subset(
+  params$run_id,
+  params$run_id_year,
+  analyses_paths, TRUE
+)
+
+list2env(data_new, envir = .GlobalEnv)
+
+rm(data_new)
+
+comparison_paths <- list(
+  output = list(
+    list(
+      s3 = base_paths$output$assessment_card$s3,
+      key = "assessment_card"
+    ),
+    list(
+      s3 = base_paths$output$assessment_pin$s3,
+      key = "assessment_pin"
+    ),
+    list(
+      s3 = base_paths$output$metadata$s3,
+      key = "metadata"
+    ),
+    list(
+      s3 = base_paths$output$performance_test$s3,
+      key = "performance_test"
+    ),
+    list(
+      s3 = base_paths$output$shap$s3,
+      key = "shap"
+    )
+  )
+)
+
+data_comparison <- model_fetch_run_subset(
+  params$comparison_run_id,
+  params$comparison_run_id_year,
+  comparison_paths, TRUE
+)
+
+list2env(data_comparison, envir = .GlobalEnv)
+
+rm(data_comparison)
+
+all_vars <- ls()
+
+rename_var <- function(var_name, suffix, new_suffix) {
```

Comment:
This I'd move into a helper file location, but it's here for continuity. It removes the run_id appendage, and replaces it with new / comparison. For the sake of this report, it makes it much more readable throughout.